Explorer/Assets/DCL/InWorldCamera/CameraReelGallery/Components/ReelThumbnailView.cs

```diff
@@ -0,0 +1,39 @@
+using DCL.UI;
+using System;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+namespace DCL.InWorldCamera.CameraReelGallery.Components
+{
+    public class ReelThumbnailView : MonoBehaviour, IPointerEnterHandler, IPointerExitHandler, IDisposable
+    {
+        [Header("References")]
+        public Image thumbnailImage;
```

Comment:
Why it does not follow the same `SerializedField` principle? You could also you a property to a private setter marked by `[field: SerializeField]`